apps/remix-ide-e2e/src/tests/staticAnalysis.test.ts

```diff
@@ -43,7 +43,8 @@ function runTests (browser: NightwatchBrowser) {
     .waitForElementPresent('#staticanalysisresult .warning', 2000, true, function () {
       listSelectorContains(['Use of tx.origin',
         'Fallback function of contract TooMuchGas requires too much gas',
-        'TooMuchGas.() : Variables have very similar names "test" and "test1".'],
+        'TooMuchGas.() : Variables have very similar names "test" and "test1".',
+        'TooMuchGas.() : Variables have very similar names "test" and "test1".' ],
```

Comment:
why is that duplicated?

Reply:
Because now we get warning for each similar variable name usage. So we will get one warning for `test` and another for `test1`